src/features/common/Layout/CustomModal/index.tsx

```diff
@@ -0,0 +1,58 @@
+import Modal from '@mui/material/Modal';
+import React from 'react';
+import i18next from '../../../../../i18n';
+import styles from './CustomModal.module.scss';
+
+interface Props {
+  isOpen: boolean;
+  onClick1: () => void;
+  onClick2: () => void;
+  buttonTitle: string;
+  modalTitle: string;
+  modalSubtitle: string;
+}
+
+const { useTranslation } = i18next;
+
+export default function CustomModal({
+  isOpen,
+  onClick1,
+  onClick2,
+  buttonTitle,
+  modalTitle,
+  modalSubtitle,
+}: Props) {
+  const { t } = useTranslation(['editProfile']);
+  return (
+    <Modal open={isOpen} hideBackdrop className={'modalContainer'}>
+      <div className={styles.modal}>
+        <div className={styles.modal__titleContainer}>
+          <div className={styles.modal__titleText}>
+            {' '}
+            <b> {modalTitle} </b>
+          </div>
+          <div className={styles.modal__subtitle}>{modalSubtitle}</div>
+        </div>
+        <div
+          className={styles.buttonContainer}
+          style={{ justifyContent: 'center' }}
+        >
+          <button
+            id={'Continue'}
+            className={styles.continueButton}
+            onClick={onClick1}
```

Comment:
@prachigarg19 Please give it a meaningful name, maybe something like `handleContinue`